WordPress/Classes/Models/Media.swift

```diff
@@ -9,14 +9,14 @@ extension Media {
     /// Increments the AutoUpload failure count for this Media object.
     ///
     @objc
-    func incrementAutoUploadFailureCount() {
+    public func incrementAutoUploadFailureCount() {
```

Comment:
This and the other changes were necessary for Keystone to compile. See how https://buildkite.com/automattic/wordpress-ios/builds/27253/steps?sid=01961f84-fc91-442d-89c1-84f330f7b000 failed.